COFFLoader.c

```diff
@@ -117,18 +121,18 @@ void* process_symbol(char* symbolstring) {
     HMODULE llHandle = NULL;
 #endif
 
-    memcpy(localcopy, symbolstring, strlen(symbolstring));
```

Comment:
Bonus fix: if `symbolstring` were over 1024 characters for whatever reason, it'd overflow `localcopy`.